cmd/krew/cmd/install.go

```diff
@@ -85,10 +85,23 @@ Remarks:
 				return errors.New("--archive can be specified only with --manifest")
 			}
 
+			installed, err := installation.ListInstalledPlugins(paths.InstallReceiptsPath())
+			if err != nil {
+				return errors.Wrap(err, "failed to find all installed versions")
+			}
+			skipped := false
 			var install []index.Plugin
 			for _, name := range pluginNames {
+				if _, ok := installed[name]; ok {
+					fmt.Fprintf(os.Stderr, "plugin \"%s\" is already installed\n", name)
+					skipped = true
```

Comment:
I'm not sure what this part does. We already have the skipping behavior implemented (see below), we can't implement it again like this. I don't see how this changes anything?

```
$ kubectl krew install krew
Updated the local copy of plugin index.
Installing plugin: krew
W1209 19:07:29.190869   69309 install.go:124] Skipping plugin "krew", it is already installed
```

Reply:
I guess it helps us not look up the deleted manifest of a plugin that's already installed. I don't think we need this, the error you added below does a proper job.

Reply:
I thought that skipping the manifest lookup for an already installed plugin and displaying a different message was what we wanted for this based on your comment [here](https://github.com/kubernetes-sigs/krew/issues/421#issuecomment-563102188):

> ...  we try to load manifest from fs again, I wonder why. we should be treating it like ErrAlreadyInstalled and say it's already installed

If its not necessary I can totally remove it, but I thought it seemed a little nicer.

Another reason I added it was for the case when the plugin(s) that are being installed don't exist so that the help message doesn't get printed. It felt out of place to show that as well.

Reply:
Yeah, I think it's making the code unnecessarily more complex and it solves a small problem that rarely occurs if ever (installing an installed plugin directly by name, after its manifest is which occurred after installing it for the first time). Let's abandon this.

Reply:
Sounds good, I'll remove it